tests/cases/fourslash/codeFixClassImplementInterfaceComments.ts

```diff
@@ -36,9 +36,9 @@ verify.codeFix({
         /**close-brace prefix*/ }
 /**close-brace prefix*/ }
 class C implements N.I {
-    /** property prefix */ a /** colon prefix */: N.E.a;
```

Comment:
This was accidentally changed in #20338 to include the comments -- but we really shouldn't be copying the comments over from the base class. The change to `helpers.ts` makes us no longer copy the comments, and is a basically independent change to the rest of this PR. Discovered this because the presence of a comment made the formatter no longer insert indentation for some reason.